trainee/preetbista/flightcrud/src/main/java/com/flight/flightcrud/dto/PaymentInfoDto.java

```diff
@@ -1,7 +1,9 @@
 package com.flight.flightcrud.dto;
 
 import com.flight.flightcrud.model.PassengerInfo;
+import lombok.AllArgsConstructor;
```

Comment:
Better to remove unnecessary imports, as it imports the .class files of the imported while executing.